src/csv.hpp

```diff
@@ -0,0 +1,1248 @@
+// Copyright: (2012-2015) Ben Strasser <code@ben-strasser.net>
```

Comment:
It was actually 2 minutes slower than regular std::getline! I talked about it here: https://github.com/mapbox/directions/issues/501#issuecomment-297209471